atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/OracleDbTimestampBoundStore.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.keyvalue.dbkvs.timestamp;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Preconditions;
+import com.palantir.atlasdb.AtlasDbConstants;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+import com.palantir.nexus.db.pool.ConnectionManager;
+
+public final class OracleDbTimestampBoundStore extends AbstractDbTimestampBoundStore {
+    private static final Logger log = LoggerFactory.getLogger(OracleDbTimestampBoundStore.class);
+
+    private static final String ORACLE_ALREADY_EXISTS_ERROR = "ORA-00955";
+
+    public static OracleDbTimestampBoundStore create(ConnectionManager connManager) {
+        OracleDbTimestampBoundStore oracleDbTimestampBoundStore = new OracleDbTimestampBoundStore(
```

Comment:
Feels a little odd to have duplicate logic here just to get a different table name, especially since it's a constant and I don't see it being changed.  I could see an argument for being at to inject this name for testing but otherwise I would want to try a refactor to have each impl have a method for getting the timestamp and then you can move the create and the constructor into the abstract class.